### PR TITLE
add remediation variable to integration variables

### PR DESCRIPTION
**Pull request by woutersm**
Added the remediation variable to the list of integration variables.

### CODE REVIEW

File: docs/programs/integration-variables.md [hunk 1]
```diff
@@ -32,3 +32,4 @@ Variable | Details | Example
 `{{title}}` | The title of the report. |
 `{{vulnerability_information}}` | The description of the vulnerability in the report. |
 `{{weakness}}` | The CWE weakness of the report. |
+`{{remediation_advice}}` | The remediation advice created from the custom remediation advice with a fall back to the automated remediation advice. |
```
Comment (martijnrusschen) [inline docs/programs/integration-variables.md:35]: Can you add it in alphabetical ordering?

Comment (martijnrusschen) [reply docs/programs/integration-variables.md:35]: Do we have more documentation on remediation advice? Perhaps link it from the description.

Comment (woutersm) [reply docs/programs/integration-variables.md:35]: [This](https://docs.hackerone.com/programs/hackbot.html) is the only place where the remediation advice is described in the docs.  But I don't think it's adding any value to link to that page, what do you think?

Comment (martijnrusschen) [reply docs/programs/integration-variables.md]: Ah no, that doesn't add any value indeed :)

**Review by martijnrusschen — APPROVED** [review]

**Comment by stacyspiva** [comment]
@martijnrusschen @woutersm The description repeats "remediation advice" 3 times, which isn't very ideal:
> The **remediation advice** created from the custom **remediation advice** with a fall back to the automated **remediation advice.** 

Can we simplify the description to something like: The customized remediation advice from MITRE. This will fall back to the automated remediation advice if there is no guidance from MITRE.

**Pull request merged**